### ISSUE TITLE
Feature Request: Listener that evaluates the network during training

### ISSUE THREAD

**Issue by ajohnsonz** [deeplearning4j/deeplearning4j#3383]
#### Issue Description

In the same way that the ScoreIterationListener can periodically print out the cost of the training set during training, it would be helpful to print out the accuracy of predictions on the test set. For example, combine the output of ScoreIterationListener with the output of new Evaluation(n).eval(), every X iterations.

#### Version Information

Not available in 0.8.0

**Comment by raver119** [comment]
Implemented & available at current master. Once snapshots will be rebuilt - it will be available there as well.
